build.sbt

```diff
@@ -810,7 +810,10 @@ lazy val `persistence-cassandra-scaladsl` = (project in file("persistence-cassan
 
 
 lazy val `persistence-jdbc-core` = (project in file("persistence-jdbc/core"))
-  .dependsOn(`persistence-core` % "compile;test->test")
+  .dependsOn(
+    `persistence-core` % "compile;test->test",
+    logback % Test
+  )
```

Comment:
There were no tests previously in `persistence-jdbc-core`, so I had to do a little refactoring.

Moved the `logback % Test` dependency from the `persistence-jdbc-javadsl` and `persistence-jdbc-scaladsl` modules to `persistence-jdbc-core`. Because those child projects already depend on this one with scope `"compile;test->test"`, they inherit the logback test dependency transitively.

The reason these depend on `logback` in the `Test` scope in the first place is to bind a concrete slf4j implementation to use when running tests, but we don't want to depend on a concrete implementation in the normal compile or runtime scope in order to allow users to choose their own.